app/views/questions/_questions.html.erb

```diff
@@ -18,7 +18,7 @@
         <% end %>
         </span>
         <p style="margin-top:30px;"><% if logged_in_as(['admin','moderator']) %><a class="btn btn-default btn-xs" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> Spam</a><% end %>
-        <a href="#" class="btn btn-default btn-xs">Post an answer</a></p>
+        <a href="<%= node.path(:question) %>#post-answer" class="btn btn-default btn-xs">Post an answer</a></p>
```

Comment:
Wierd! the Post an answer link that I added was somehow reverted in any previous changes. I added it back. Nevertheless it previously wan't correct so I put the correct one this time.